pkgs/applications/networking/instant-messengers/rambox/hide-check-for-updates.patch

```diff
@@ -1,35 +0,0 @@
-https://github.com/saenzramiro/rambox/issues/1283
```

Comment:
Now that this patch is gone, we should patch `--without-update` option: https://github.com/saenzramiro/rambox/issues/1283